repository/s.json

```diff
@@ -4843,6 +4843,15 @@
 				}
 			]
 		},
+		{
+			"details": "https://github.com/djsegal/SublimeTextGitKraken",
```

Comment:
You must not use "Sublime" in your package name per our naming conventions, since all packages for Sublime Text are … for Sublime Text (exceptions are possible but not here).